wsl/src/main/resources/META-INF/wsl-only.xml

```diff
@@ -0,0 +1,8 @@
+<idea-plugin>
```

Comment:
Why do we need a separate `wsl` module with separate XML configuration if we don't use any additional dependency?
Looks like it's worth moving all code from wsl module to core one.
It also should solve a problem with tests